helm/templates/postgres/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.postgres.enabled }}
```

Comment:
same for postgres? https://bitnami.com/stack/postgresql/helm
I guess this is an overkill?

Reply:
Yeah if deploying a postgres is only for sandbox it might be overkill.

Reply:
+1